graphql/database/users/logEmailVerified.js

```diff
@@ -29,6 +34,21 @@ const logEmailVerified = async (userContext, userId) => {
   if (userContext.emailVerified) {
     try {
       await rewardReferringUser(userContext, userId)
+      const override = getPermissionsOverride(ADMIN_MANAGEMENT)
+      const originalInvitedUsers = await InvitedUsersModel.query(
+        override,
+        userContext.email
+      )
+        .usingIndex('InvitesByInvitedEmail')
+        .execute()
+      await Promise.all(
+        originalInvitedUsers.map(invitedUserEntry =>
+          InvitedUsersModel.update(override, {
+            ...invitedUserEntry,
+            invitedId: userId,
+          })
+        )
+      )
```

Comment:
Add tests for this. (Our code coverage GitHub bot isn't working right now. Not sure why and need to fix.)